Gemfile

```diff
@@ -4,7 +4,7 @@ gem 'codecov', require: false, group: :test
 gem 'coveralls', '0.8.21', require: false
 gem 'facter', '>= 1.7.0'
 gem 'metadata-json-lint'
-gem 'oneview-sdk', '~> 5.5.0'
+gem 'oneview-sdk', '~> 5.6.0'
```

Comment:
can you rebase your branch as this change has been already merged with master.

Reply:
Thanks for pointing it out. But dont think there is a conflict and it should resolve itself if any in this case.